static/app/views/onboarding/setupDocsLoader.tsx

```diff
@@ -0,0 +1,240 @@
+      setProjectKeyUpdateError(true);
+    }
+  }, [api, location.query.product, organization.slug, project.slug, projectKey?.id]);
+
+  useEffect(() => {
+    fetchData();
+  }, [fetchData, organization.slug, project.slug]);
+
+  useEffect(() => {
+    updateSelectedProducts();
+  }, [updateSelectedProducts, location.query.product]);
+
+  return (
+    <Fragment>
+      <SetupIntroduction
+        stepHeaderText={t(
+          'Configure %s SDK',
+          platforms.find(p => p.id === currentPlatform)?.name ?? ''
+        )}
+        platform={currentPlatform}
+      />
+      <ProductSelection
+        defaultSelectedProducts={[PRODUCT.PERFORMANCE_MONITORING, PRODUCT.SESSION_REPLAY]}
+        lazyLoader
+        skipLazyLoader={close}
+      />
+
+      {projectKeyUpdateError && (
+        <LoadingError
+          message={t('Failed to update the project key with the selected products.')}
+          onRetry={() => updateSelectedProducts()}
+        />
+      )}
+
+      {!hasLoadingError ? (
+        projectKey !== null && <ProjectKeyInfo projectKey={projectKey} />
+      ) : (
+        <LoadingError
+          message={t('Failed to load Client Keys for the project.')}
+          onRetry={() => fetchData()}
+        />
+      )}
+    </Fragment>
+  );
+}
+
+function ProjectKeyInfo({projectKey}: {projectKey: ProjectKey}) {
+  const [showOptionalConfig, setShowOptionalConfig] = useState(false);
+
+  const loaderLink = projectKey.dsn.cdn;
+
+  const configCodeSnippet = beautify.js(
+    `Sentry.onLoad(function() {
+  Sentry.init({
+    // No need to configure DSN here, it is already configured in the loader script
+    // You can add any additional configuration here
+    sampleRate: 0.5,
+  })
+})`,
+    {indent_size: 2}
+  );
+
+  return (
+    <DocsWrapper>
+      <DocumentationWrapper>
+        <h2>{t('Install')}</h2>
+        <p>
+          {t(
+            'After including this script tag to a page, we’ll send a test error to make sure Sentry is set up correctly.'
+          )}
+        </p>
+
+        <CodeSnippet dark language="html" onCopy={() => {}}>
+          {beautify.html(
+            `<script src="${loaderLink}" crossorigin="anonymous"></script>`,
+            {indent_size: 2, wrap_attributes: 'force-expand-multiline'}
+          )}
+        </CodeSnippet>
+
+        <hr />
+        <OptionalConfigWrapper>
+          <ToggleButton
+            priority="link"
+            borderless
+            size="zero"
+            icon={<IconChevron direction={showOptionalConfig ? 'down' : 'right'} />}
+            aria-label={t('Toggle optional configuration')}
+            onClick={() => setShowOptionalConfig(!showOptionalConfig)}
+          />
+          <h2 onClick={() => setShowOptionalConfig(!showOptionalConfig)}>
+            {t('Configuration (Optional)')}
+          </h2>
+        </OptionalConfigWrapper>
+        {showOptionalConfig && (
+          <div>
+            <CodeSnippet dark language="js" onCopy={() => {}}>
```

Comment:
shall we update the `onCopy` function? currently it won't do anything

Reply:
I removed this!

Reply:
I find it quite useful... maybe we can follow up on this in another PR? 🙏